app/src/main/java/info/nightscout/androidaps/utils/ScrollingOffsetListener.kt

```diff
@@ -0,0 +1,24 @@
+package info.nightscout.androidaps.utils
+
+import androidx.viewpager2.widget.ViewPager2
+import com.google.android.material.appbar.AppBarLayout
+
+class ScrollingOffsetListener(
+    private val scrollView: ViewPager2
+): AppBarLayout.OnOffsetChangedListener {
+
+    private var originalHeight = 0
+    private var firstOffset = true
+
+    override fun onOffsetChanged(layout: AppBarLayout?, offset: Int) {
+        if(firstOffset) {
```

Comment:
Please apply code-formatting on the new/changed code (CTRL+ALT+L on Windows or Command+Option+L on Mac.)
If you want to use only a selection, you can use  CTRL+SHIFT+ALT+L on Windows